SwiftLSPClient/JSONRPC/ProtocolTransport.swift

```diff
@@ -132,7 +132,8 @@ public class ProtocolTransport {
             
         responder(.success(data))
 
-        responders.removeValue(forKey: message.id)
+        // This doesn't seem to be threadsafe
+        // responders.removeValue(forKey: message.id)
```

Comment:
Ok, hmm. Going to have to dig into this more. For now, how do you feel about leaving this change out of this particular PR? We can look into thread safety of this area more directly.

Reply:
Backed out the  change in another commit making Location Hashable for symbol lookup.